libmscore/mscore.h

```diff
@@ -213,7 +213,7 @@ enum class IconType : signed char {
       SBEAM, MBEAM, NBEAM, BEAM32, BEAM64, AUTOBEAM,
       FBEAM1, FBEAM2,
       VFRAME, HFRAME, TFRAME, FFRAME, MEASURE,
-      BRACKETS, PARENTHESES
+      BRACKETS, PARENTHESES, BRACES,
```

Comment:
The last comma shouldn't be here

Reply:
Doesn't harm either and leads to leaner diffs on nedt extension

Reply:
See abobe, https://github.com/musescore/MuseScore/pull/6182/files#diff-f0e7e3ffeb78101e7c88d9bb470413bcR50